app/views/pointless_feedback/messages/new.html.erb

```diff
@@ -1,3 +1,9 @@
+<% if PointlessFeedback.using_captcha? %>
+  <head>
+    <script src='https://www.google.com/recaptcha/api.js'></script>
+  </head>
+<% end %>
```

Comment:
Will this make the DOM like `<head></head>` -> `<body><head></head></body>` if using typical Rails application layout? Might move to not be in the `<head>` if so?